packages/scaffold/src/partials/w3m-input-token/index.ts

```diff
@@ -48,13 +48,13 @@ export class W3mInputToken extends LitElement {
         text=${this.token.symbol}
         imageSrc=${this.token.iconUrl}
         @click=${this.handleSelectButtonClick.bind(this)}
-        >Select token</wui-token-button
```

Comment:
Is this ok?

Reply:
Yeap, the w3m-input-token doesn't getting children